packages/core/src/node/console-logger-server.ts

```diff
@@ -59,7 +64,22 @@ export class ConsoleLoggerServer implements ILoggerServer {
     async log(name: string, logLevel: number, message: string, params: any[]): Promise<void> {
         const configuredLogLevel = await this.getLogLevel(name);
         if (logLevel >= configuredLogLevel) {
-            ConsoleLogger.log(name, logLevel, message, params);
+            const fullMessage = ConsoleLogger.log(name, logLevel, message, params);
+            this.logToFile(fullMessage, params);
```

Comment:
ConsoleLogger.log provides the formatter message. Why must the params be passed down and reformat with utils again?

Reply:
`ConsoleLogger.log` doesn't include the parameters into the message, but passes them into `console.<method>`. `util.format` will then create a human readable string from message + parameters.